lib/src/num_ext.dart

```diff
@@ -0,0 +1,87 @@
+/// Int extensions
+extension IntExt on int {
+  /// Returns ordinal of [int]
+  /// ``` dart
+  /// print(1.ordinal); // 1st
+  /// print(22.ordinal); // 22nd
+  /// print(143.ordinal); // 143rd
+  /// print(0.ordinal); // 0th
+  /// print((-1).ordinal); // -1st
+  /// print(12.ordinal); // 12th
+  /// print(69.ordinal); // 69th
+  /// ```
+  String get ordinal {
+    if (this < 0) {
+      return '-${(-this).ordinal}';
+    }
+    if (this % 100 >= 11 && this % 100 <= 13) {
+      return '${this}th';
+    }
+    switch (this % 10) {
+      case 1:
+        return '${this}st';
+      case 2:
+        return '${this}nd';
+      case 3:
+        return '${this}rd';
+      default:
+        return '${this}th';
+    }
+  }
+
+  /// Returns roman number representation of [int] from 1 to 3999
+  /// ``` dart
+  /// print(12.toRoman); // XII
+  /// print(455.toRoman); // CDLV
+  /// print(1.toRoman); // I
+  /// print(3999.toRoman); // MMMCMXCIX
+  /// ```
+  String get toRoman {
```

Comment:
Use a function instead of a getter, because the name is `toRoman`.
If you want to keep a getter name it without the `to`.